src/resources/SearchUsageMetrics/LicenseMetrics/LicenseMetricsInterface.ts

```diff
@@ -0,0 +1,35 @@
+export interface RestListOfMetricsModel {
+    metrics: RestMetricsModel[];
+}
+
+export interface RestMetricsModel {
+    id: string;
+    label: string;
+}
+
+export interface RestListOfMetricValues {
+    values: RestMetricValueForDate[];
```

Comment:
Just to be sure: should we include a JSDoc to explain every properties of every interface?

Reply:
I've added JSDoc for parameters to put in calls